docs/installation/manual/installing_fedora_syn_and_blazegraph.md

```diff
@@ -50,194 +47,97 @@ grant all privileges on database FEDORA_DB to FEDORA_DB_USER;
-<xml = 'http://www.w3.org/XML/1998/namespace'>
-<xmlns = 'http://www.w3.org/2000/xmlns/'>
-<xs = 'http://www.w3.org/2001/XMLSchema'>
-<xsi = 'http://www.w3.org/2001/XMLSchema-instance'>
+The basics of these configuration files have been pulled largely from the templates in Islandora-Devops/islandora-playbook [internal Fedora role](https://github.com/Islandora-Devops/islandora-playbook/tree/dev/roles/internal/Islandora-Devops.fcrepo); you may consider referencing the playbook’s templates directory for more details.
+
+#### Namespace prefixes
+
+`i8_namespaces.yml` is a list of namespaces used by Islandora that may not necessarily be present in Fedora; we add them here to ensure we can use them in queries.
+
+`/opt/fcrepo/config/i8_namespaces.yml | tomcat:tomcat/644`
+```{ .yaml .copy }
+# Islandora 8/Fedora namespaces
+#
+# This file contains ALL the prefix mappings, if a URI
+# does not appear in this file it will be displayed as 
+# the full URI in Fedora. 
+acl: http://www.w3.org/ns/auth/acl#
+bf: http://id.loc.gov/ontologies/bibframe/
+cc: http://creativecommons.org/ns#
+dc: http://purl.org/dc/elements/1.1/
+dcterms: http://purl.org/dc/terms/
+dwc: http://rs.tdwg.org/dwc/terms/
+ebucore: http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#
+exif: http://www.w3.org/2003/12/exif/ns#
+fedoraconfig: http://fedora.info/definitions/v4/config#
+fedoramodel: info:fedora/fedora-system:def/model#
+foaf: http://xmlns.com/foaf/0.1/
+geo: http://www.w3.org/2003/01/geo/wgs84_pos#
+gn: http://www.geonames.org/ontology#
+iana: http://www.iana.org/assignments/relation/
+islandorarelsext: http://islandora.ca/ontology/relsext#
+islandorarelsint: http://islandora.ca/ontology/relsint#
+ldp: http://www.w3.org/ns/ldp#
+memento: http://mementoweb.org/ns#
+nfo: http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#
+ore: http://www.openarchives.org/ore/terms/
+owl: http://www.w3.org/2002/07/owl#
+premis: http://www.loc.gov/premis/rdf/v1#
+prov: http://www.w3.org/ns/prov#
+rdf: http://www.w3.org/1999/02/22-rdf-syntax-ns#
+rdfs: http://www.w3.org/2000/01/rdf-schema#
+rel: http://id.loc.gov/vocabulary/relators/
+schema: http://schema.org/
+skos: http://www.w3.org/2004/02/skos/core#
+test: info:fedora/test/
+vcard: http://www.w3.org/2006/vcard/ns#
+webac: http://fedora.info/definitions/v4/webac#
+xml: http://www.w3.org/XML/1998/namespace
+xmlns: http://www.w3.org/2000/xmlns/
+xs: http://www.w3.org/2001/XMLSchema
+xsi: http://www.w3.org/2001/XMLSchema-instance
 ```
 
-We intend to have Crayfish installed later. Since Fedora needs to be able to read data from Crayfish, we need to tell Fedora that the Crayfish endpoint is a valid data source.
+#### Allowed External Content Hosts
 
-`/opt/fcrepo/config/allowed_hosts.txt | tomcat:tomcat/644`
-```
-http://CRAYFISH_HOST:CRAYFISH_PORT/
+We have Fedora provide metadata for some resources that are contained in Drupal. Fedora needs to know to allow access to these External Content hosts.
```

Comment:
If this has anything to do with crayfish (i.e. if you have to change it if you change your Crayfish URLs) then mentioning Crayfish here is a good thing. 

But if it really only has to do with drupal resources then this is good.

Reply:
This is only Drupal because of the External Content resources created.

Reply:
Sorry, it's confusing because the stuff removed doesn't look related to the stuff added. Or is it? For the same file, we used to be talking about that line in terms of Crayfish but now it's about external content (I wonder if this means Remote Video, of if there is another "external content" meant here?)

Also, sorry.  Like below, in the stuff added, can you format the file like the others in this section?

Reply:
I'm not really clear on the original content of this document. 

Allowed External Content hosts are required when making a Fedora resource that copies, proxies or redirects a resource on some other website. The hosts in this file is the hostname of this other website. 

My understanding is that the only place we currently do this is some of the derivatives that are stored in Drupal. So the original file is stored directly in Fedora via the Flysystem connector and then Crayfish generates some derivatives, but these derivatives are then stored in Drupal and there is a Fedora resource made that points to this resource in Drupal.

So to my mind, this was incorrect before. But it would depend on your specific setup.

Reply:
OK, that makes total sense, thank you!